generate-stackbrew-library.sh

```diff
@@ -10,33 +10,77 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
 versions=( */ )
 versions=( "${versions[@]%/}" )
-url='git://github.com/docker-library/golang'
 
-echo '# maintainer: InfoSiftr <github@infosiftr.com> (@infosiftr)'
-echo '# maintainer: Johan Euphrosine <proppy@google.com> (@proppy)'
+# get the most recent commit which modified any of "$@"
+fileCommit() {
+	git log -1 --format='format:%H' HEAD -- "$@"
+}
+
+# get the most recent commit which modified any of "$1/Dockerfile" or any file COPY'd from "$1/Dockerfile"
+dirCommit() {
+	local dir="$1"; shift
+	(
+		cd "$dir"
+		fileCommit \
+			Dockerfile \
+			$(git show "$ref":./Dockerfile | awk '
+				toupper($1) == "COPY" {
+					for (i = 2; i < NF; i++) {
+						print $i
+					}
+				}
+			')
+	)
+}
+
+self="$(basename "$BASH_SOURCE")"
+cat <<-EOH
+# this file is generated via https://github.com/docker-library/golang/blob/$(fileCommit "$self")/$self
+
+Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
+             Joseph Ferguson <yosifkit@gmail.com> (@yosifkit),
+             Johan Euphrosine <proppy@google.com> (@proppy)
+GitRepo: https://github.com/docker-library/golang.git
+
+EOH
+
+# prints "Tags: $1, $2, $3, ..."
+outputTags() {
+	echo "Tags:$(printf ', %s' "$@" | cut -d, -f2-)"
```

Comment:
Yeah, I did a similar hack in my attempt: `$(IFS=, ; echo "${tags[*]/#/ }")`

i.e. Prepend each tag with a space, then set `IFS` to a comma to join the output with an extra space at the beginning